CHANGEME.md

```diff
@@ -355,4 +355,7 @@ You take one down, you pass it around, 32 bottles of rum on the wall.
 32 bottles of rum on the wall, 32 bottles of rum!  
 You take one down, you pass it around, 31 bottles of rum on the wall.
 
+30 bottles of rum on the wall, 30 bottles of rum!  
+You take one down, you pass it around, 31 bottles of rum on the wall.
```

Comment:
Ahoy, @otakdark! Thank you for your active contribution to the project. This new verse goes a different way, the song follows the pattern of N bottles to N - 1 per verse. Please adjust bottle count ☺️🙏